problems/2.functions.js

```diff
@@ -2,36 +2,40 @@
 * @param {string} name - The name of the user to greet
 * @returns {null}
 */
-function logGreeting() {
-
+function greeting(name) {
+  let str = null
+  return str
 }
+console.log(greeting())
 
 /** Returns a custom greeting string in the form "Hello Avery!"
 * @param {string} name - The name of the user to greet
 * @returns {string} - The custom greeting
 */
-function greeting() {
-
+function greeting(name) {
+  let str = "Hello"
+  return str + name
```

Comment:
This is close! Just need to check your tests and see what's different. Need the "!" at the end. and some spacing between the words.